examples/with-apivideo-upload/.env.development

```diff
@@ -0,0 +1 @@
+API_KEY=
```

Comment:
I think this should be changed to `.env.local.example` instead as `.env.development` is not gitignored.